.github/workflows/build.yml

```diff
@@ -3,29 +3,32 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
-name: Go
+name: build
 on: [push, pull_request]
 jobs:
 
   unitTest:
-    name: Unit Test
+    name: Unit test
     runs-on: ${{ matrix.os }}
+    timeout-minutes: 5
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest]
     steps:
 
-    - name: Set up Go 1.13
+    - name: Setup Go 1.13
       uses: actions/setup-go@v1
       with:
         go-version: 1.13
       id: go
 
     - uses: actions/checkout@v1
 
-    - name: Unit Test
-      timeout-minutes: 5
+    - name: Run unit test
```

Comment:
Add `timeout-minutes: 5` as well

Reply:
timeout goes with the job not the step AFAIK.
https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes

(this was the change I made).

Reply:
The documentation is unclear on what the default step timeout is...
https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes

Reply:
We will need to revisit the timeouts. I just copied 10mins into each long step and job.